app/src/main/kotlin/com/wire/android/migration/MigrationMapper.kt

```diff
@@ -56,18 +62,23 @@ class MigrationMapper @Inject constructor() {
                 access = mapAccess(access),
                 accessRole = listOf(TEAM_MEMBER, NON_TEAM_MEMBER, SERVICE),
                 removedBy = null,
-                lastReadDate = LocalDateTime.MIN.toString(),
-                lastModifiedDate = LocalDateTime.MIN.toString(),
-                lastNotificationDate = LocalDateTime.MIN.toString(),
+                lastReadDate = DateTimeUtil.currentIsoDateTimeString(),
```

Comment:
Can't we take these values from the scala db? When we set the current time as `lastReadDate` then all unread messages will be marked as read when we migrate, right?

Reply:
good point i will migrate this value too